addon/tailwind/components/bourbon-button.css

```diff
@@ -17,81 +17,81 @@
   @apply .bourbon-border-transparent;
 }
 
-.bourbon-button--icon-only {
+.BourbonButton-icon {
   @apply .bourbon-p-2;
   @apply .bourbon-min-w-button-icon;
   @apply .bourbon-min-h-button-icon;
   @apply .bourbon-border-mercury;
 }
 
-.bourbon-button--icon-only svg {
+.BourbonButton-icon svg {
   width: 20px;
   height: 20px;
 }
 
-.bourbon-button--icon-text svg {
+.BourbonButton-iconText svg {
   width: 16px;
   height: 16px;
 }
 
-.bourbon-primary-button {
+.BourbonButton--primary {
   @apply .bourbon-text-white;
   @apply .bourbon-bg-fern;
   @apply .bourbon-border-fern;
 }
 
-.bourbon-delete-button {
+.BourbonButton--delete {
   @apply .bourbon-text-white;
   @apply .bourbon-bg-red;
   @apply .bourbon-border-red;
 }
 
-.bourbon-primary-button:hover {
+.BourbonButton--primary:hover {
   @apply .bourbon-bg-emerald;
   @apply .bourbon-border-emerald;
 }
 
-.bourbon-primary-button:active,
-.bourbon-primary-button:focus {
+.BourbonButton--primary:active,
+.BourbonButton--primary:focus {
   @apply .bourbon-bg-fruit-salad;
   @apply .bourbon-border-fruit-salad;
   box-shadow: inset 0 0 0 1px var(--white);
   outline: none;
 }
 
-.bourbon-secondary-button {
+.BourbonButton--secondary {
   @apply .bourbon-text-mine-shaft;
   @apply .bourbon-bg-alabaster;
   @apply .bourbon-border-mercury;
 }
 
-.bourbon-secondary-button:hover {
+.BourbonButton--secondary:hover {
   @apply .bourbon-bg-white;
 }
 
-.bourbon-secondary-button:active,
-.bourbon-secondary-button:focus {
+.BourbonButton--secondary:active,
+.BourbonButton--secondary:focus {
   @apply .bourbon-bg-concrete;
   @apply .bourbon-border-concrete;
   box-shadow: inset 0 0 0 1px var(--dodger-blue);
   outline: none;
 }
 
-.bourbon-primary-button.bourbon-disable-button {
+.BourbonButton--primary.BourbonButton--disable {
```

Comment:
Assuming this is a primary button that's disabled, the end should probably be `disabled` instead of `disable`.

Reply:
makes sense! 👍